synapse/handlers/room.py

```diff
@@ -876,39 +876,71 @@ class RoomListHandler(BaseHandler):
 
     @defer.inlineCallbacks
     def get_public_room_list(self):
-        chunk = yield self.store.get_rooms(is_public=True)
```

Comment:
``get_rooms`` is only called in one place now, and that is to get the number of rooms in synapse.app.homeserver:phone_stats_home. It might be nice to kill it for good.